commands-yml/commands/session/delete.yml

```diff
@@ -15,7 +15,7 @@ example_usage:
 
   javascript_wdio:
     |
-      driver.endSession();
+      driver.deleteSession();
```

Comment:
I don't think this is right. 

https://github.com/admc/wd/search?q=deleteSession&unscoped_q=deleteSession

[wd](https://github.com/admc/wd/) has `driver.quit()` for ending the session.

Reply:
Sorry. I thought this was `wd`, not `webdriverio`. Ignore this.